### PR TITLE
Remove obsolete location medadata in NGSIv1

### PR DESCRIPTION
**Pull request by fgalan**
- [x] Remove support in code
- [x] deprecated.md
- [x] Check & fix script
- [x] ~Migrate script to Python 3~ (we are going to use at the end entities_consistency.py script + info in release notes)
- [x] ~Check & fix script documentation~ (we are going to use at the end entities_consistency.py script + info in release notes)
- [x] ~Check & fix script and documentation definitive Orion version (currently we use "3.5.0" as placeholder)~ (we are going to use at the end entities_consistency.py script + info in release notes)
- [x] CNR

### CODE REVIEW

**Comment by fgalan** [comment]
https://github.com/telefonicaid/fiware-orion/compare/3c188d...27d4e5a show the code simplification we can achieve with the removal. It can be even higher if the "FIXME PR" mark is also taken into account.

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR does some modifications to the English documentation. It would be great if you could sync the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I sent the PR #4492. Thanks.

File: src/lib/mongoBackend/location.cpp [hunk 1]
```diff
@@ -319,37 +321,6 @@ static bool getGeoJson
...
-    geoJson->append("type", "Point");
-
-    orion::BSONArrayBuilder ba;
-    ba.append(aLong);
-    ba.append(aLat);
-    geoJson->append("coordinates", ba.arr());
-
-    return true;
-  }
-
```
Comment (fgalan) [inline src/lib/mongoBackend/location.cpp:352]: This seems to be a big block of duplicated code. The changes done PR https://github.com/telefonicaid/fiware-orion/pull/4491 confirm and fix it.

NTC (post-merge comment)

File: src/lib/ngsi/ContextElementResponse.cpp [hunk 2]
```diff
@@ -125,7 +125,119 @@ ContextElementResponse::ContextElementResponse
...
+      caP->creDate = getNumberFieldF(attr, ENT_ATTRS_CREATION_DATE);
+    }
+
+    if (attr.hasField(ENT_ATTRS_MODIFICATION_DATE))
+    {
+      caP->modDate = getNumberFieldF(attr, ENT_ATTRS_MODIFICATION_DATE);
+    }
+
+    entity.attributeVector.push_back(caP);
+  }
```
Comment (fgalan) [inline src/lib/ngsi/ContextElementResponse.cpp:240]: This seems to be a big block of duplicated code. The changes done PR https://github.com/telefonicaid/fiware-orion/pull/4491 confirm and fix it.

NTC (post-merge comment)

**Pull request merged**